positionChecker/positionCheck.py

```diff
@@ -1,12 +1,174 @@
+
+        reprojectdst, _ = cv2.projectPoints(self.reprojectsrc, rotation_vec, translation_vec, self.cam_matrix,
+                                            self.dist_coeffs)
+
+        reprojectdst = tuple(map(tuple, reprojectdst.reshape(8, 2)))
+
+        # calc euler angle
+        rotation_mat, _ = cv2.Rodrigues(rotation_vec)
+        pose_mat = cv2.hconcat((rotation_mat, translation_vec))
+        _, _, _, _, _, _, euler_angle = cv2.decomposeProjectionMatrix(pose_mat)
+        #"""yaw = euler_angle[1], roll = euler_angle[2], pitch = euler_angle[0]"""
+
+        
+        s=''
+        i=0
+        pit = [0,0,0,0,0,0,0]
+
+        j=i%7
+        i+=1;
+
+        #print(i)
+        pit[j]=euler_angle[0]
+        avg=0
+        
+        
+        if(j==0):  
+            for k in range(7):
+                avg+=pit[k]
+            if(avg/7>2 and avg/7<=17):
+                s= "Humped Back Sitting."
+                
+            elif(avg/7<-2 and avg/7>=-17):
+                s = 'Inclined Sitting Position.'
+            
+            elif(avg/7>17):
+                s = 'You are looking down.'
+                
+            elif(avg/7<-17):
+                s = 'You are overly inclined.'
+            
+            else:
+                s = "You are approximatly sitting straight."
+        print(s)
+
+        return [reprojectdst, euler_angle, s]
+    
+   
+
+    def check_position(self, image):
+        # return
+
+        ### LOADS IN TEMPORARY PHOTO SAMPLE ####
+        #img = cv2.imread(cv2.samples.findFile("temp_photo.jpg"))
+        #if img is None:
+        #    sys.exit("Could not read the image.")
+       
+        #print(image)
+
+        #image = Image.open('image.jpg')
+
+        ## this opens up image popup #######
+        #image.show()
+
+        frame = cv2.cvtColor(np.array(image), cv2.COLOR_RGB2BGR)
+
+        #print(image) 
+        
+        # cv2.imshow('image', frame)
+        # cv2.waitKey(0)
+        face_rects = self.detector(frame, 0)
+        print(face_rects)
+                
+        if (len(face_rects)==0):
+            print("I can't see you")
+                    #if( s=='You are looking down.' or s== 'Humped Back Sitting.'):
+                    #    print("extremely down.")
+                    #elif(s=='Inclined Sitting Position.' or s=='You are overly inclined.'):
+                    #    print("You are overly inclined.")
+                    #print("WHEN FACE_RECT == 0: " + s)
+
+        response = "no image detected"                
+        if len(face_rects) > 0:
+            print("greater than 0")
+            shape = self.predictor(frame, face_rects[0])
+            shape = face_utils.shape_to_np(shape)
+            """if 'shape' is locals():
+                print('we got it')"""
+                    
+            poseArray = self.get_head_pose(shape)
+            response = poseArray[2]
+
+        #cv2.imshow("Display window", img)
+        #k = cv2.waitKey(0)
```

Comment:
Example Comment: why is this commented, can you clean up and remove the comments or does it need to be here?